packages/sources/dxfeed-test/src/endpoint/price-ws.ts

```diff
@@ -0,0 +1,157 @@
+import { WebSocketTransport } from '@chainlink/external-adapter-framework/transports/websocket'
+import { makeLogger, SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { WebSocket } from '@chainlink/external-adapter-framework/transports'
+import { customSettings } from '../config'
+
+const logger = makeLogger('DxFeed Websocket')
+
+export type DXFeedMessage = {
+  channel: string
+  clientId?: string
+  id: string
+  data: [string, [string, number, number, number, number, string, number, string, number, number]]
+  successful?: boolean
+  advice?: {
+    interval: number
+    timeout: number
+    reconnect: string
+  }
+}[]
+
+export type EndpointTypes = {
+  Request: {
+    Params: { base: string }
+  }
+  Response: SingleNumberResultResponse
+  CustomSettings: typeof customSettings
+  Provider: {
+    WsMessage: DXFeedMessage
+  }
+}
+
+const META_HANDSHAKE = '/meta/handshake'
+const META_CONNECT = '/meta/connect'
+const SERVICE_SUB = '/service/sub'
+const SERVICE_DATA = '/service/data'
+
+const tickerIndex = 0
+const priceIndex = 6
+
+class DxFeedWebsocketTransport extends WebSocketTransport<EndpointTypes> {
+  private _connectionClientId = ''
+  id = 1
+
+  get connectionClientId() {
+    return this._connectionClientId
+  }
+
+  set connectionClientId(id) {
+    this._connectionClientId = id
+  }
+  get handshakeMsg() {
```

Comment:
Would be great with some spacing here and between the other functions

```suggestion
  }
  
  get handshakeMsg() {
```